### PR TITLE
Only wrap as much code with System Lambda as necessary

### PR DESCRIPTION
**Pull request by stefanbirkner**
This makes it clear which part of the code needs the environment
variables.

### CODE REVIEW

**Review by nasirov — CHANGES_REQUESTED** [review]
Minor code style remarks

File: src/test/java/nasirov/yv/proxy/server/task/SocketServerThreadTest.java [hunk 1]
```diff
@@ -52,24 +52,27 @@ public static void tearDown() {
 	@Test
 	@SneakyThrows
 	public void shouldReturnOkResponse() {
+		//given
+		Socket clientSocket = new Socket("localhost", SOCKET_SERVER_PORT);
+
```
[the remark this answers is not in the document]
Comment (stefanbirkner) [reply src/test/java/nasirov/yv/proxy/server/task/SocketServerThreadTest.java]: Done.

File: src/test/java/nasirov/yv/proxy/server/task/SocketServerThreadTest.java [hunk 2]
```diff
@@ -52,24 +52,27 @@ public static void tearDown() {
...
 					out.flush();
 					waitForAnotherThread();
-					//then
-					String result = getResponse(clientSocket);
-					System.out.println("result:\n" + result);
-					assertTrue(result.contains("HTTP/1.1 200 OK"));
-					assertTrue(result.contains("body"));
-					out.close();
 				});
+
```
[the remark this answers is not in the document]
Comment (stefanbirkner) [reply src/test/java/nasirov/yv/proxy/server/task/SocketServerThreadTest.java]: Done

File: src/test/java/nasirov/yv/proxy/server/task/SocketServerThreadTest.java [hunk 3]
```diff
@@ -52,24 +52,27 @@ public static void tearDown() {
...
+		//then
+		String result = getResponse(clientSocket);
+		out.close();
+
```
[the remark this answers is not in the document]
Comment (stefanbirkner) [reply src/test/java/nasirov/yv/proxy/server/task/SocketServerThreadTest.java]: Done

**Review by nasirov — APPROVED** [review]

**Pull request merged**